pkg/component/component.go

```diff
@@ -245,6 +245,12 @@ func CreateFromPath(client *occlient.Client, params occlient.CreateArgs) error {
 		return err
 	}
 
+	if params.Wait {
+		podSelector := fmt.Sprintf("deploymentconfig=%s-%s", labels["app.kubernetes.io/component-name"], labels["app"])
```

Comment:
This would be a good place to add the spinner.

For example, you'd use:

```go
s:= log.Spinner("Waiting for Pod to come up")
defer s.End(false)
```